ruma-federation-api/src/openid/get_userinfo/v1.rs

```diff
@@ -0,0 +1,26 @@
+//! [GET /_matrix/federation/v1/openid/userinfo](https://matrix.org/docs/spec/server_server/r0.1.4#get-matrix-federation-v1-openid-userinfo)
+
+use ruma_api::ruma_api;
+use ruma_identifiers::UserId;
+
+ruma_api! {
+    metadata: {
+        description: "Exchanges an OpenID access token for information about the user who generated the token. Currently this only exposes the Matrix User ID of the owner.",
+        method: GET,
+        name: "openid_userinfo",
```

Comment:
We like to set the names to the same name as the module.
```suggestion
        name: "get_openid_userinfo",
```

Reply:
@iinuwa The module is named just `get_userinfo` though ;)

But thanks for spotting this!